CityOfDetroit/bloom
# 973/favorites on dashboard

- Closes [#issue](https://github.com/CityOfDetroit/bloom/issues/973)
- Addresses #issue
depends on: https://github.com/CityOfDetroit/bloom/pull/1089

## Description
This exposes favorites on the dashboard on the public side

## How Can This Be Tested/Reviewed?
create a user on the public site (or create a new user)
log in as that user
navigate to the user's dashboard, and notice there is a favorites button
click on the favorites button
notice there is text letting the user know they have not favorited anything yet
navigate back to the listings list
favorite a few listings
go back to the dashboard
click into the favorites
notice the listings you favorited are in the new list
unfavorite a few listings and refresh the page
notice the unfavorited listings are gone

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
